addons/misc/CfgWeapons.hpp

```diff
@@ -16,4 +16,43 @@ class CfgWeapons
             scope = 0;
         };
     };
+
+    class ItemCore;
+    class kat_Armband_Red_Cross_Item: ItemCore {
+
+        detectRange = -1;
+        author = "Miss Heda, vccv9040 (Swedish Forces Pack)";
+        simulation = "ItemMineDetector";
+        displayName = CSTRING(Armband_Red_Cross_ItemName);
+        scope = 2;
+        cost = 2;
+        side = 3;
```

Comment:
and this one is only seen by me in config of backpacks on wiki page, are you sure these two are required? 🤔 
ok, checked in config viewer, ItemCore dont even have anything like side or cost (at least i dont see it)